src/data.ts

```diff
@@ -589,12 +592,13 @@ const protocols: Protocol[] = [
     portfolioUrl: "https://www.usefelix.xyz/points",
     defillama: {
       slug: "felix-usdhl",
-      description: "USDhl is a fiat stable custom-built for the Hyperliquid ecosystem, serving as a building block for HIP-3, fx, payments, and more",
+      description:
+        "USDhl is a fiat stable custom-built for the Hyperliquid ecosystem, serving as a building block for HIP-3, fx, payments, and more",
       logo: "https://icons.llama.fi/felix-usdhl.jpg",
       twitter: "usd_hl",
-      url: "https://www.usefelix.xyz?ref=4BF702FF"
-    }
-  }
+      url: "https://www.usefelix.xyz?ref=4BF702FF",
```

Comment:
This is defillama's ref link we can remove this and replace with ours (in `referralUrl`)
```suggestion
      url: "https://www.usefelix.xyz",
```